lxd/seccomp.go

```diff
@@ -799,7 +807,7 @@ func CallForkmknod(c container, dev types.Device, requestPID int, permissionsOnl
 		"forksyscall", "mknod", dev["pid"], dev["path"],
 		dev["mode_t"], dev["dev_t"], dev["hostpath"],
 		fmt.Sprintf("%d", uid), fmt.Sprintf("%d", gid),
-		fmt.Sprintf("%d", deBool()))
+		fmt.Sprintf("%d", BoolToInt(permissionsOnly)))
```

Comment:
we only do this in two places, why not just make it `permissionsOnly == 1` which would be identical to what the function does and possibly a bit more readable too?

Reply:
Fixed.